api/src/pcapi/admin/custom_views/venue_view.py

```diff
@@ -394,7 +394,6 @@ def update_view(self) -> Response:
         url = get_redirect_target() or self.get_url(".index_view")
         change_form = VenueChangeForm(request.form)
         if change_form.validate():
-
             venue_ids: list[int] = list(map(int, change_form.ids.data.split(",")))
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏C'est une majeure de black ?
Ce changement de règle doit faire fail black dans tous les repos où c'est utilisé :thinking:

Reply:
Oui. On est passé de la 22.12.0 début décembre à la 23.1.0 il y a sept heures.